src/main/java/com/capitalone/creditocr/model/ingest/LetterQueueProcessor.java

```diff
@@ -0,0 +1,206 @@
+            descriptions.put("favor", queueType.GENERAL);
+            descriptions.put("please consider", queueType.GENERAL);
+            descriptions.put("guarded my account", queueType.GENERAL);
+            descriptions.put("maintained this account appropriately", queueType.GENERAL);
+            descriptions.put("good customer", queueType.GENERAL);
+            descriptions.put("occasional", queueType.GENERAL);
+            descriptions.put("negative notations", queueType.GENERAL);
+            descriptions.put("guarded our agreement", queueType.GENERAL);
+            descriptions.put("scattered late pay", queueType.GENERAL);
+            descriptions.put("bearing my signature", queueType.GENERAL);
+            descriptions.put("scattered negative", queueType.GENERAL);
+            descriptions.put("scattered", queueType.GENERAL);
+            descriptions.put("managed to accumulate", queueType.GENERAL);
+            descriptions.put("managed to accrue", queueType.GENERAL);
+            descriptions.put("accumulated seldom", queueType.GENERAL);
+            descriptions.put("managed to acquire", queueType.GENERAL);
+            descriptions.put("minor late notations", queueType.GENERAL);
+            descriptions.put("seldom negative", queueType.GENERAL);
+
+            descriptions.put("inquiry", queueType.INQUIRIES);
+            descriptions.put("inquiries", queueType.INQUIRIES);
+            descriptions.put("i did not apply for credit", queueType.INQUIRIES);
+            descriptions.put("unauthorized view of credit", queueType.INQUIRIES);
+            descriptions.put("didn't authorize", queueType.INQUIRIES);
+            descriptions.put("visit a dealership", queueType.INQUIRIES);
+        }
+    }
+
+    /**
+     * Removes punctuation and makes all letters lower case
+     * @param letterdata Letter containing the text to process
+     * @return String containing lowercase text without punctuation
+     */
+    public String normalizeLetterText(LetterData letterdata) {
+        return letterdata.getText().toLowerCase().replaceAll("\\p{IsPunctuation}", "");
+    }
+
+    /**
+     * Main method which will fill the queueCount map with the appropriate number of found instances for a particular key
+     * @param normalizedText This must be a String run through the normalizeLetterText() method.
+     */
+    public void processText(String normalizedText) {
+        // Check to make sure all maps have already been initialized
+        this.initialize();
+
+        // Go through each element in the map and check if the string is in the text
+        for (HashMap.Entry<String, Enum> entry : descriptions.entrySet()) {
+            // This performance is bad and I would love a better way.
+            // If the text has the key words, add 1 to the count of it's respective queue.
+            if (normalizedText.contains(entry.getKey())) {
+                Integer count = queueCount.get(entry.getValue());
```

Comment:
In general, it's better to use the primitive `int`, instead of `Integer`, as you get better performance. Not a huge deal though.